cal-automation-server-recording/BlobStorageHelper.ts

```diff
@@ -0,0 +1,86 @@
+import { BlobSASPermissions, BlobSASSignatureValues, generateBlobSASQueryParameters, StorageSharedKeyCredential } from "@azure/storage-blob";
+import { Logger, MessageType } from './Logger';
+
+const { BlobServiceClient } = require('@azure/storage-blob');
+const fs = require("fs");
+
+export class BlobStorageHelper {
+  static containerClientUri: string;
+
+  public static async uploadFileToStorage(containerName: string,
+    blobName: string,
+    blobConnectionString: string) {
+    try {
+      const blobServiceClient = BlobServiceClient.fromConnectionString(blobConnectionString)
+      const containerClient = blobServiceClient.getContainerClient(containerName);
+      var containerExist = await containerClient.exists();
+      if (!containerExist) {
+        var output = 'Blob Container -> ' + containerName + ' is unavailable'
+        var error = { 'output': output, 'statusCode': '400' }
+        Logger.logMessage(MessageType.ERROR, output)
+        return error;
+      }
+
+      const blockBlobClient = containerClient.getBlockBlobClient(blobName);
+
+      var blobExist = await blockBlobClient.exists();
+      if (blobExist) {
+        var output = 'Blob  -> ' + blobName + ' is already present'
+        var error = { 'output': output, 'statusCode': '400' }
+        Logger.logMessage(MessageType.ERROR, output)
+        return error;
+      }
+      else {
+        try {
```

Comment:
we already having try at outer block, and we do not do anything differently so this can be removed.

Reply:
done.